apps/barista/src/components/live-example/live-example.ts

```diff
@@ -153,4 +221,34 @@ export class BaLiveExample implements OnDestroy {
       );
     }
   }
+
+  /**
+   * Updates value of behavior subject after
+   * successful copy to clipboard action.
+   */
+  private _copySuccess(): void {
+    this._copied = true;
+    this._timerSubscription = timer(1000).subscribe(
+      () => (this._copied = false),
+    );
+  }
+
+  /**
+   * Transforms given code by replacing empty lines,
+   * and wrapping each line in a span element.
+   */
+  private _enhanceCode(code: string, type: BaSourceType): string {
+    // Remove empty lines at the start of the source
+    let transformedCode = code.replace(/^(\s*\r?\n)*/g, '');
```

Comment:
NIT: For those regexes, I would add a comment with an example regex101 to showcase the regex